hack/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+IN=origin
+for version in v5.5 v5.6; do
```

Comment:
That sounds dangerous. We already have a versions variable in Makefile, I'll use that.